From 7aabf53c8b26c933d6c80c6196cd8740edd42ea7 Mon Sep 17 00:00:00 2001
From: Edward Wang <ewang@cyngn.com>
Date: Mon, 13 Apr 2015 14:17:00 -0700
Subject: [PATCH 34/51] FM: reenable radio text visibility after station info
 updated

After radio station info updated, radio text view is set to GONE.
We need turn it back on if FM is still enabled.

Change-Id: I7acea564a352693e19d9e6d934d641327b90be7e
(cherry picked from commit 086a510710b073b4b5c5cd4a0f72341dc6690b84)
---
 fmapp2/src/com/caf/fmradio/FMRadio.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/fmapp2/src/com/caf/fmradio/FMRadio.java b/fmapp2/src/com/caf/fmradio/FMRadio.java
index c383266..3709849 100644
--- a/fmapp2/src/com/caf/fmradio/FMRadio.java
+++ b/fmapp2/src/com/caf/fmradio/FMRadio.java
@@ -1686,7 +1686,6 @@ public class FMRadio extends Activity
                 bStatus = mService.fmOn();
                 if(bStatus) {
                    tuneRadio(FmSharedPreferences.getTunedFrequency());
-                   enableRadioOnOffUI();
                 }else {
                    Log.e(LOGTAG, "mService.fmOn failed");
                    mCommandFailed = CMD_FMON;
@@ -2631,6 +2630,7 @@ public class FMRadio extends Activity
              mTunedStation.setPI(0);
              mTunedStation.setPty(0);
              updateStationInfoToUI();
+             enableRadioOnOffUI();
          }catch (RemoteException e) {
             e.printStackTrace();
          }
@@ -2688,6 +2688,7 @@ public class FMRadio extends Activity
       mERadioTextScroller.stopScroll();
       mUpdatePickerValue = true;
       updateStationInfoToUI();
+      enableRadioOnOffUI();
    }
 
    Runnable mRadioEnabled = new Runnable() {
-- 
2.17.1

